### ISSUE TITLE
Calculation of residual enthalpy of liquid

### ISSUE THREAD

**Issue by Jackxi1024** [zmeri/PC-SAFT#88]
Hello! I found that pc-saft is very accurate in calculating the properties of gas and the density of liquid, but the calculation of residual enthalpy of liquid is different from Aspen.

The parameter "phase" is needed in pcsaft_den() to distinguish between liquid and vapor. Why pcsaft_ hres() has no parameter "phase" ? I think this is the reason why the calculation of liquid residual enthalpy is different from Aspen.

Looking forward to your reply, thank you very much

**Comment by zmeri** [comment]
The density is one of the input parameters to pcsaft_hres(), and therefore,
the phase doesn't need to be specified. If you want the residual enthalpy
of the liquid phase, then you give the liquid density. Same for the vapor
phase.

I have tested the residual calorific property values and found that they
are quite similar to those from Aspen, but I only tested for a handful of
compounds. Let me know how it goes.
